frontend/pages/admin/AppSettingsPage/AppSettingsPage.tsx

```diff
@@ -63,15 +55,10 @@ const AppSettingsPage = (): JSX.Element => {
         })
         .finally(() => {
           refetchConfig();
-          // Config must be updated in both Redux and AppContext
-          dispatch(getConfig())
-            .then((configState: IConfig) => {
-              setConfig(configState);
-            })
-            .catch(() => false);
+          appConfig && setConfig(appConfig);
```

Comment:
Will this be a race if `refechConfig` does not complete before line 58 runs? What do you think about doing `setConfig` with the `onSuccess` callback of the `useQuery` hook?

Reply:
Great catch! Fixed.